pkg/providers/instance/aksmachineinstancetimestamputils.go

```diff
@@ -0,0 +1,61 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package instance
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// AKSMachineTimestampFormat is the RFC3339 format used for AKS machine creation timestamp tags
+	AKSMachineTimestampFormat = "2006-01-02T15:04:05.00Z"
+)
```

Comment:
Same as `time.RFC3339` but with 2 decimal digits for franctional seconds - may be worth calling out (as it explains some of what follows)

Also, does not need to be exported